src/store/getters.js

```diff
@@ -43,6 +43,10 @@ const TESTNET_ASSETS = [
   })
 }, {})
 
+const mapLegacyProvidersToSupported = {
+  oneinchV3: 'oneinchV4'
```

Comment:
Do we need `oneinchV3` at all now? Is `oneinchV4` supposed to completely replace `oneinchV3` ?

Reply:
Yes, they are doing the same and V3 will be deprecated soon . 

I need this line, because we have swaps in the history that have been using V3 and they are in the local storage. 
If we do not have a backward support this will result in visualization issues in the old swaps